benchmarks/grpo/grpo_local/run_engine.py

```
@@ -0,0 +1,78 @@
+        "--max-steps", str(args.steps),
+        "--group-size", "4",
+        "--lora-rank", "16",
+        "--lora-alpha", "16",
+        "--learning-rate", "1e-4",
+        "--clip-epsilon", "0.2",
+        "--kl-coef", "0.0",
+        "--reward-funcs", "ground_truth:field=ground_truth",
+        "--output-json", "/tmp/grpo_bench_metrics.json",
+    ]
+
+    proc = subprocess.run(run_cmd, capture_output=True, text=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)